src/domUpdates.js

```diff
@@ -5,13 +5,31 @@ today: null,
 allTrips: null,
 allDestinations: null,
 
-applyGlobals(currentTraveler, date, trips, destinations) {
+accessGlobalInfo(currentTraveler, date, trips, destinations) {
   this.currentTraveler = currentTraveler;
   this.today = date;
   this.allTrips = trips;
   this.allDestinations = destinations;
 },
 
+onLoadFire() {
+  this.displayCurrentTraveler(this.currentTraveler);
+  this.displayTrips(this.currentTraveler, 'upcoming');
+  this.displayExpenses(this.currentTraveler.calculateTripMoneySpentInYear());
+  this.createDateStopInCalendar();
+  this.destinationDropDownForBooking();
+},
+
+createDateStopInCalendar() {
```

Comment:
I may chat with you about this. I haven't dealt with dates yet, but I was considering using date js